### PR TITLE
Fix licenses

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

**Comment by wu-sheng** [comment]
OMG, so many MIT licenses. @kezhenxu94 This seems a pretty good example and try for your new skywalking-eyes 0.2.0

**Comment by wu-sheng** [comment]
@mrproliu Could you share the skywalking-eyes scan result? I think we don't add check mechanism yet, so, please provide the result table.

File: dist/licenses/LICENSE-mysql.txt [hunk 1]
```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```
Comment (wu-sheng) [inline dist/licenses/LICENSE-mysql.txt:1]: @mrproliu Please be clear about this is dependency. `LICENSE-mysql` seems very strange, it looks like mysql server license, which is GPL, but at here, it is MPL v2 only.

Recommend to use `LICENSE-gosqldriver-mysql`

**Comment by mrproliu** [comment]
> @mrproliu Could you share the skywalking-eyes scan result? I think we don't add check mechanism yet, so, please provide the result table.

Here is using skywalking-eyes dependency resolve result, these results help me to quickly recognize the identity of the license.

|Dependency                                              |                       License|
|--------------------------------------------------------|------------------------------|
|github.com/apache/skywalking-infra-e2e                  |                    Apache-2.0|
|bazil.org/fuse                                          |                  BSD-3-Clause|
|cloud.google.com/go                                     |                    Apache-2.0|
|cloud.google.com/go/bigquery                            |                    Apache-2.0|
|cloud.google.com/go/datastore                           |                    Apache-2.0|
|cloud.google.com/go/firestore                           |                    Apache-2.0|
|cloud.google.com/go/pubsub                              |                    Apache-2.0|
|cloud.google.com/go/storage                             |                    Apache-2.0|
|dmitri.shuralyov.com/gpu/mtl                            |                  BSD-3-Clause|
|github.com/Azure/azure-sdk-for-go                       |                    Apache-2.0|
|github.com/Azure/go-ansiterm                            |                           MIT|
|github.com/Azure/go-autorest                            |                    Apache-2.0|
|github.com/Azure/go-autorest/autorest                   |                    Apache-2.0|
|github.com/Azure/go-autorest/autorest/adal              |                    Apache-2.0|
|github.com/Azure/go-autorest/autorest/date              |                    Apache-2.0|
|github.com/Azure/go-autorest/autorest/mocks             |                    Apache-2.0|
|github.com/Azure/go-autorest/logger                     |                    Apache-2.0|
|github.com/Azure/go-autorest/tracing                    |                    Apache-2.0|
|github.com/BurntSushi/toml                              |                           MIT|
|github.com/BurntSushi/xgb                               |                  BSD-3-Clause|
|github.com/MakeNowJust/heredoc                          |                           MIT|
|github.com/Microsoft/go-winio                           |                           MIT|
|github.com/Microsoft/hcsshim                            |                           MIT|
|github.com/Microsoft/hcsshim/test                       |                           MIT|
|github.com/NYTimes/gziphandler                          |                    Apache-2.0|
|github.com/OneOfOne/xxhash                              |                    Apache-2.0|
|github.com/PuerkitoBio/purell                           |                  BSD-3-Clause|
|github.com/PuerkitoBio/urlesc                           |                  BSD-3-Clause|
|github.com/Shopify/logrus-bugsnag                       |                           MIT|
|github.com/alecthomas/template                          |                  BSD-3-Clause|
|github.com/alecthomas/units                             |                           MIT|
|github.com/alessio/shellescape                          |                           MIT|
|github.com/armon/circbuf                                |                           MIT|
|github.com/armon/consul-api                             | MPL-2.0-no-copyleft-exception|
|github.com/armon/go-metrics                             |                           MIT|
|github.com/armon/go-radix                               |                           MIT|
|github.com/asaskevich/govalidator                       |                           MIT|
|github.com/aws/aws-sdk-go                               |                    Apache-2.0|
|github.com/beorn7/perks                                 |                           MIT|
|github.com/bgentry/speakeasy                            |                           MIT|
|github.com/bitly/go-simplejson                          |                           MIT|
|github.com/bketelsen/crypt                              |                           MIT|
|github.com/blang/semver                                 |                           MIT|
|github.com/bshuster-repo/logrus-logstash-hook           |                           MIT|
|github.com/buger/jsonparser                             |                           MIT|
|github.com/bugsnag/bugsnag-go                           |                           MIT|
|github.com/bugsnag/panicwrap                            |                           MIT|
|github.com/cenkalti/backoff                             |                           MIT|
|github.com/census-instrumentation/opencensus-proto      |                    Apache-2.0|
|github.com/cespare/xxhash                               |                           MIT|
|github.com/cespare/xxhash/v2                            |                           MIT|
|github.com/chai2010/gettext-go                          |                  BSD-3-Clause|
|github.com/checkpoint-restore/go-criu/v4                |                    Apache-2.0|
|github.com/chzyer/readline                              |                           MIT|
|github.com/chzyer/test                                  |                           MIT|
|github.com/cilium/ebpf                                  |                           MIT|
|github.com/client9/misspell                             |                           MIT|
|github.com/cncf/udpa/go                                 |                    Apache-2.0|
|github.com/cockroachdb/datadriven                       |                    Apache-2.0|
|github.com/containerd/aufs                              |                    Apache-2.0|
|github.com/containerd/btrfs                             |                    Apache-2.0|
|github.com/containerd/cgroups                           |                    Apache-2.0|
|github.com/containerd/console                           |                    Apache-2.0|
|github.com/containerd/containerd                        |                    Apache-2.0|
|github.com/containerd/continuity                        |                    Apache-2.0|
|github.com/containerd/fifo                              |                    Apache-2.0|
|github.com/containerd/go-cni                            |                    Apache-2.0|
|github.com/containerd/go-runc                           |                    Apache-2.0|
|github.com/containerd/imgcrypt                          |                    Apache-2.0|
|github.com/containerd/nri                               |                    Apache-2.0|
|github.com/containerd/ttrpc                             |                    Apache-2.0|
|github.com/containerd/typeurl                           |                    Apache-2.0|
|github.com/containerd/zfs                               |                    Apache-2.0|
|github.com/containernetworking/cni                      |                    Apache-2.0|
|github.com/containernetworking/plugins                  |                    Apache-2.0|
|github.com/containers/ocicrypt                          |                    Apache-2.0|
|github.com/coreos/bbolt                                 |                           MIT|
|github.com/coreos/etcd                                  |                    Apache-2.0|
|github.com/coreos/go-iptables                           |                    Apache-2.0|
|github.com/coreos/go-oidc                               |                    Apache-2.0|
|github.com/coreos/go-semver                             |                    Apache-2.0|
|github.com/coreos/go-systemd                            |                    Apache-2.0|
|github.com/coreos/go-systemd/v22                        |                    Apache-2.0|
|github.com/coreos/pkg                                   |                    Apache-2.0|
|github.com/cpuguy83/go-md2man/v2                        |                           MIT|
|github.com/creack/pty                                   |                           MIT|
|github.com/cyphar/filepath-securejoin                   |                  BSD-3-Clause|
|github.com/d2g/dhcp4                                    |                  BSD-3-Clause|
|github.com/d2g/dhcp4client                              | MPL-2.0-no-copyleft-exception|
|github.com/d2g/dhcp4server                              | MPL-2.0-no-copyleft-exception|
|github.com/d2g/hardwareaddr                             | MPL-2.0-no-copyleft-exception|
|github.com/davecgh/go-spew                              |                           ISC|
|github.com/denverdino/aliyungo                          |                    Apache-2.0|
|github.com/dgrijalva/jwt-go                             |                           MIT|
|github.com/dgryski/go-sip13                             |                           MIT|
|github.com/docker/distribution                          |                    Apache-2.0|
|github.com/docker/docker                                |                    Apache-2.0|
|github.com/docker/go-connections                        |                    Apache-2.0|
|github.com/docker/go-events                             |                    Apache-2.0|
|github.com/docker/go-metrics                            |                    Apache-2.0|
|github.com/docker/go-units                              |                    Apache-2.0|
|github.com/docker/libtrust                              |                    Apache-2.0|
|github.com/docker/spdystream                            |                    Apache-2.0|
|github.com/docopt/docopt-go                             |                           MIT|
|github.com/dustin/go-humanize                           |                           MIT|
|github.com/elazarl/goproxy                              |                  BSD-3-Clause|
|github.com/emicklei/go-restful                          |                           MIT|
|github.com/envoyproxy/go-control-plane                  |                    Apache-2.0|
|github.com/envoyproxy/protoc-gen-validate               |                    Apache-2.0|
|github.com/evanphx/json-patch                           |                  BSD-3-Clause|
|github.com/exponent-io/jsonpath                         |                           MIT|
|github.com/fatih/camelcase                              |                           MIT|
|github.com/fatih/color                                  |                           MIT|
|github.com/form3tech-oss/jwt-go                         |                           MIT|
|github.com/fsnotify/fsnotify                            |                  BSD-3-Clause|
|github.com/fullsailor/pkcs7                             |                           MIT|
|github.com/fvbommel/sortorder                           |                           MIT|
|github.com/ghodss/yaml                                  |                           MIT|
|github.com/go-gl/glfw                                   |                  BSD-3-Clause|
|github.com/go-gl/glfw/v3.3/glfw                         |                  BSD-3-Clause|
|github.com/go-ini/ini                                   |                    Apache-2.0|
|github.com/go-kit/kit                                   |                           MIT|
|github.com/go-logfmt/logfmt                             |                           MIT|
|github.com/go-logr/logr                                 |                    Apache-2.0|
|github.com/go-openapi/jsonpointer                       |                    Apache-2.0|
|github.com/go-openapi/jsonreference                     |                    Apache-2.0|
|github.com/go-openapi/spec                              |                    Apache-2.0|
|github.com/go-openapi/swag                              |                    Apache-2.0|
|github.com/go-redis/redis                               |                  BSD-2-Clause|
|github.com/go-sql-driver/mysql                          |                       MPL-2.0|
|github.com/go-stack/stack                               |                           MIT|
|github.com/godbus/dbus                                  |                  BSD-2-Clause|
|github.com/godbus/dbus/v5                               |                  BSD-2-Clause|
|github.com/gogo/googleapis                              |                    Apache-2.0|
|github.com/golang/glog                                  |                    Apache-2.0|
|github.com/golang/groupcache                            |                    Apache-2.0|
|github.com/golang/mock                                  |                    Apache-2.0|
|github.com/golang/protobuf                              |                  BSD-3-Clause|
|github.com/golangplus/bytes                             |                  BSD-3-Clause|
|github.com/golangplus/fmt                               |                  BSD-3-Clause|
|github.com/golangplus/testing                           |                  BSD-3-Clause|
|github.com/google/btree                                 |                    Apache-2.0|
|github.com/google/go-cmp                                |                  BSD-3-Clause|
|github.com/google/gofuzz                                |                    Apache-2.0|
|github.com/google/martian                               |                    Apache-2.0|
|github.com/google/pprof                                 |                    Apache-2.0|
|github.com/google/renameio                              |                    Apache-2.0|
|github.com/google/uuid                                  |                  BSD-3-Clause|
|github.com/googleapis/gax-go/v2                         |                  BSD-3-Clause|
|github.com/googleapis/gnostic                           |                    Apache-2.0|
|github.com/gopherjs/gopherjs                            |                  BSD-2-Clause|
|github.com/gorilla/handlers                             |                  BSD-2-Clause|
|github.com/gorilla/mux                                  |                  BSD-3-Clause|
|github.com/gorilla/websocket                            |                  BSD-2-Clause|
|github.com/gregjones/httpcache                          |                           MIT|
|github.com/grpc-ecosystem/go-grpc-middleware            |                    Apache-2.0|
|github.com/grpc-ecosystem/go-grpc-prometheus            |                    Apache-2.0|
|github.com/grpc-ecosystem/grpc-gateway                  |                  BSD-3-Clause|
|github.com/hashicorp/consul/api                         | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/consul/sdk                         | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/errwrap                            | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go-cleanhttp                       |                       MPL-2.0|
|github.com/hashicorp/go-immutable-radix                 | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go-msgpack                         |                  BSD-3-Clause|
|github.com/hashicorp/go-multierror                      | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go-rootcerts                       | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go-sockaddr                        | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go-syslog                          |                           MIT|
|github.com/hashicorp/go-uuid                            | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/go.net                             |                  BSD-3-Clause|
|github.com/hashicorp/golang-lru                         | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/hcl                                | MPL-2.0-no-copyleft-exception|
|github.com/hashicorp/logutils                           |                       MPL-2.0|
|github.com/hashicorp/mdns                               |                           MIT|
|github.com/hashicorp/memberlist                         |                       MPL-2.0|
|github.com/hashicorp/serf                               | MPL-2.0-no-copyleft-exception|
|github.com/ianlancetaylor/demangle                      |                  BSD-3-Clause|
|github.com/imdario/mergo                                |                  BSD-3-Clause|
|github.com/inconshreveable/mousetrap                    |                    Apache-2.0|
|github.com/j-keck/arping                                |                           MIT|
|github.com/jessevdk/go-flags                            |                  BSD-3-Clause|
|github.com/jmespath/go-jmespath                         |                    Apache-2.0|
|github.com/jonboulle/clockwork                          |                    Apache-2.0|
|github.com/json-iterator/go                             |                           MIT|
|github.com/jstemmer/go-junit-report                     |                           MIT|
|github.com/jtolds/gls                                   |                           MIT|
|github.com/julienschmidt/httprouter                     |                  BSD-3-Clause|
|github.com/kisielk/errcheck                             |                           MIT|
|github.com/kisielk/gotool                               |                           MIT|
|github.com/klauspost/compress                           |                  BSD-3-Clause|
|github.com/konsorten/go-windows-terminal-sequences      |                           MIT|
|github.com/kr/pretty                                    |                           MIT|
|github.com/kr/pty                                       |                           MIT|
|github.com/kr/text                                      |                           MIT|
|github.com/liggitt/tabwriter                            |                  BSD-3-Clause|
|github.com/lithammer/dedent                             |                           MIT|
|github.com/mailru/easyjson                              |                           MIT|
|github.com/marstr/guid                                  |                           MIT|
|github.com/mattn/go-colorable                           |                           MIT|
|github.com/mattn/go-isatty                              |                           MIT|
|github.com/mattn/go-runewidth                           |                           MIT|
|github.com/mattn/go-shellwords                          |                           MIT|
|github.com/matttproud/golang_protobuf_extensions        |                    Apache-2.0|
|github.com/miekg/pkcs11                                 |                  BSD-3-Clause|
|github.com/mistifyio/go-zfs                             |                    Apache-2.0|
|github.com/mitchellh/cli                                |                       MPL-2.0|
|github.com/mitchellh/go-homedir                         |                           MIT|
|github.com/mitchellh/go-testing-interface               |                           MIT|
|github.com/mitchellh/go-wordwrap                        |                           MIT|
|github.com/mitchellh/gox                                | MPL-2.0-no-copyleft-exception|
|github.com/mitchellh/iochan                             |                           MIT|
|github.com/mitchellh/mapstructure                       |                           MIT|
|github.com/moby/sys/mount                               |                    Apache-2.0|
|github.com/moby/sys/mountinfo                           |                    Apache-2.0|
|github.com/moby/sys/symlink                             |                    Apache-2.0|
|github.com/moby/term                                    |                    Apache-2.0|
|github.com/modern-go/concurrent                         |                    Apache-2.0|
|github.com/modern-go/reflect2                           |                    Apache-2.0|
|github.com/morikuni/aec                                 |                           MIT|
|github.com/mrunalp/fileutils                            |                    Apache-2.0|
|github.com/munnerz/goautoneg                            |                  BSD-3-Clause|
|github.com/mwitkow/go-conntrack                         |                    Apache-2.0|
|github.com/mxk/go-flowrate                              |                  BSD-3-Clause|
|github.com/ncw/swift                                    |                           MIT|
|github.com/oklog/ulid                                   |                    Apache-2.0|
|github.com/olekukonko/tablewriter                       |                           MIT|
|github.com/onsi/ginkgo                                  |                           MIT|
|github.com/onsi/gomega                                  |                           MIT|
|github.com/opencontainers/go-digest                     |                    Apache-2.0|
|github.com/opencontainers/image-spec                    |                    Apache-2.0|
|github.com/opencontainers/runc                          |                    Apache-2.0|
|github.com/opencontainers/runtime-spec                  |                    Apache-2.0|
|github.com/opencontainers/runtime-tools                 |                    Apache-2.0|
|github.com/opencontainers/selinux                       |                    Apache-2.0|
|github.com/pelletier/go-toml                            |                           MIT|
|github.com/peterbourgon/diskv                           |                           MIT|
|github.com/pkg/errors                                   |                  BSD-2-Clause|
|github.com/pmezard/go-difflib                           |                  BSD-3-Clause|
|github.com/posener/complete                             |                           MIT|
|github.com/pquerna/cachecontrol                         |                    Apache-2.0|
|github.com/prometheus/client_golang                     |                    Apache-2.0|
|github.com/prometheus/client_model                      |                    Apache-2.0|
|github.com/prometheus/common                            |                    Apache-2.0|
|github.com/prometheus/procfs                            |                    Apache-2.0|
|github.com/prometheus/tsdb                              |                    Apache-2.0|
|github.com/rogpeppe/fastuuid                            |                  BSD-3-Clause|
|github.com/rogpeppe/go-internal                         |                  BSD-3-Clause|
|github.com/ryanuber/columnize                           |                           MIT|
|github.com/safchain/ethtool                             |                    Apache-2.0|
|github.com/satori/go.uuid                               |                           MIT|
|github.com/sean-/seed                                   |                           MIT|
|github.com/shurcooL/sanitized_anchor_name               |                           MIT|
|github.com/sirupsen/logrus                              |                           MIT|
|github.com/smartystreets/assertions                     |                           MIT|
|github.com/smartystreets/goconvey                       |                           MIT|
|github.com/soheilhy/cmux                                |                    Apache-2.0|
|github.com/spaolacci/murmur3                            |                  BSD-3-Clause|
|github.com/spf13/afero                                  |                    Apache-2.0|
|github.com/spf13/cast                                   |                           MIT|
|github.com/spf13/cobra                                  |                    Apache-2.0|
|github.com/spf13/jwalterweatherman                      |                           MIT|
|github.com/spf13/pflag                                  |                  BSD-3-Clause|
|github.com/spf13/viper                                  |                           MIT|
|github.com/stefanberger/go-pkcs11uri                    |                    Apache-2.0|
|github.com/stretchr/objx                                |                           MIT|
|github.com/stretchr/testify                             |                           MIT|
|github.com/subosito/gotenv                              |                           MIT|
|github.com/syndtr/gocapability                          |                  BSD-2-Clause|
|github.com/tchap/go-patricia                            |                           MIT|
|github.com/testcontainers/testcontainers-go             |                           MIT|
|github.com/tmc/grpc-websocket-proxy                     |                           MIT|
|github.com/ugorji/go                                    |                           MIT|
|github.com/urfave/cli                                   |                           MIT|
|github.com/vishvananda/netlink                          |                    Apache-2.0|
|github.com/vishvananda/netns                            |                    Apache-2.0|
|github.com/willf/bitset                                 |                  BSD-3-Clause|
|github.com/xeipuuv/gojsonpointer                        |                    Apache-2.0|
|github.com/xeipuuv/gojsonreference                      |                    Apache-2.0|
|github.com/xeipuuv/gojsonschema                         |                    Apache-2.0|
|github.com/xiang90/probing                              |                           MIT|
|github.com/xordataexchange/crypt                        |                           MIT|
|github.com/yuin/goldmark                                |                           MIT|
|github.com/yvasiyarov/go-metrics                        |                  BSD-2-Clause|
|github.com/yvasiyarov/gorelic                           |                  BSD-2-Clause|
|github.com/yvasiyarov/newrelic_platform_go              |                  BSD-2-Clause|
|go.etcd.io/bbolt                                        |                           MIT|
|go.etcd.io/etcd                                         |                    Apache-2.0|
|go.mozilla.org/pkcs7                                    |                           MIT|
|go.opencensus.io                                        |                    Apache-2.0|
|go.uber.org/atomic                                      |                           MIT|
|go.uber.org/multierr                                    |                           MIT|
|go.uber.org/zap                                         |                           MIT|
|golang.org/x/crypto                                     |                  BSD-3-Clause|
|golang.org/x/exp                                        |                  BSD-3-Clause|
|golang.org/x/image                                      |                  BSD-3-Clause|
|golang.org/x/lint                                       |                  BSD-3-Clause|
|golang.org/x/mobile                                     |                  BSD-3-Clause|
|golang.org/x/mod                                        |                  BSD-3-Clause|
|golang.org/x/net                                        |                  BSD-3-Clause|
|golang.org/x/oauth2                                     |                  BSD-3-Clause|
|golang.org/x/sync                                       |                  BSD-3-Clause|
|golang.org/x/sys                                        |                  BSD-3-Clause|
|golang.org/x/term                                       |                  BSD-3-Clause|
|golang.org/x/text                                       |                  BSD-3-Clause|
|golang.org/x/time                                       |                  BSD-3-Clause|
|golang.org/x/tools                                      |                  BSD-3-Clause|
|golang.org/x/xerrors                                    |                  BSD-3-Clause|
|google.golang.org/api                                   |                  BSD-3-Clause|
|google.golang.org/appengine                             |                    Apache-2.0|
|google.golang.org/cloud                                 |                    Apache-2.0|
|google.golang.org/genproto                              |                    Apache-2.0|
|google.golang.org/grpc                                  |                    Apache-2.0|
|google.golang.org/protobuf                              |                  BSD-3-Clause|
|gopkg.in/airbrake/gobrake.v2                            |                  BSD-3-Clause|
|gopkg.in/alecthomas/kingpin.v2                          |                           MIT|
|gopkg.in/cheggaaa/pb.v1                                 |                  BSD-3-Clause|
|gopkg.in/errgo.v2                                       |                  BSD-3-Clause|
|gopkg.in/fsnotify.v1                                    |                  BSD-3-Clause|
|gopkg.in/gemnasium/logrus-airbrake-hook.v2              |                           MIT|
|gopkg.in/ini.v1                                         |                    Apache-2.0|
|gopkg.in/natefinch/lumberjack.v2                        |                           MIT|
|gopkg.in/resty.v1                                       |                           MIT|
|gopkg.in/square/go-jose.v2                              |                    Apache-2.0|
|gopkg.in/yaml.v2                                        |                    Apache-2.0|
|gopkg.in/yaml.v3                                        |                MIT and Apache|
|gotest.tools                                            |                    Apache-2.0|
|gotest.tools/v3                                         |                    Apache-2.0|
|honnef.co/go/tools                                      |                           MIT|
|k8s.io/api                                              |                    Apache-2.0|
|k8s.io/apimachinery                                     |                    Apache-2.0|
|k8s.io/apiserver                                        |                    Apache-2.0|
|k8s.io/cli-runtime                                      |                    Apache-2.0|
|k8s.io/client-go                                        |                    Apache-2.0|
|k8s.io/code-generator                                   |                    Apache-2.0|
|k8s.io/component-base                                   |                    Apache-2.0|
|k8s.io/component-helpers                                |                    Apache-2.0|
|k8s.io/cri-api                                          |                    Apache-2.0|
|k8s.io/gengo                                            |                    Apache-2.0|
|k8s.io/klog                                             |                    Apache-2.0|
|k8s.io/klog/v2                                          |                    Apache-2.0|
|k8s.io/kube-openapi                                     |                    Apache-2.0|
|k8s.io/kubectl                                          |                    Apache-2.0|
|k8s.io/kubernetes                                       |                    Apache-2.0|
|k8s.io/metrics                                          |                    Apache-2.0|
|k8s.io/utils                                            |                    Apache-2.0|
|rsc.io/binaryregexp                                     |                  BSD-3-Clause|
|rsc.io/quote/v3                                         |                  BSD-3-Clause|
|rsc.io/sampler                                          |                  BSD-3-Clause|
|sigs.k8s.io/apiserver-network-proxy/konnectivity-client |                    Apache-2.0|
|sigs.k8s.io/kind                                        |                    Apache-2.0|
|sigs.k8s.io/kustomize                                   |                    Apache-2.0|
|sigs.k8s.io/structured-merge-diff/v3                    |                    Apache-2.0|
|sigs.k8s.io/structured-merge-diff/v4                    |                    Apache-2.0|
|sigs.k8s.io/yaml                                        |                           MIT|
|github.com/alexflint/go-filemutex                       |                       Unknown|
|github.com/bmizerany/assert                             |                       Unknown|
|github.com/bugsnag/osext                                |                       Unknown|
|github.com/chzyer/logex                                 |                       Unknown|
|github.com/daviddengcn/go-colortext                     |                       Unknown|
|github.com/dnaeon/go-vcr                                |                       Unknown|
|github.com/evanphx/json-patch/v5                        |                       Unknown|
|github.com/garyburd/redigo                              |                       Unknown|
|github.com/gogo/protobuf                                |                       Unknown|
|github.com/hpcloud/tail                                 |                       Unknown|
|github.com/kr/logfmt                                    |                       Unknown|
|github.com/magiconair/properties                        |                       Unknown|
|github.com/miekg/dns                                    |                       Unknown|
|github.com/mitchellh/osext                              |                       Unknown|
|github.com/pascaldekloe/goe                             |                       Unknown|
|github.com/russross/blackfriday                         |                       Unknown|
|github.com/russross/blackfriday/v2                      |                       Unknown|
|github.com/seccomp/libseccomp-golang                    |                       Unknown|
|gopkg.in/check.v1                                       |                       Unknown|
|gopkg.in/inf.v0                                         |                       Unknown|
|gopkg.in/tomb.v1                                        |                       Unknown|

ERROR failed to identify the licenses of following packages (21):
github.com/alexflint/go-filemutex
github.com/bmizerany/assert
github.com/bugsnag/osext
github.com/chzyer/logex
github.com/daviddengcn/go-colortext
github.com/dnaeon/go-vcr
github.com/evanphx/json-patch/v5
github.com/garyburd/redigo
github.com/gogo/protobuf
github.com/hpcloud/tail
github.com/kr/logfmt
github.com/magiconair/properties
github.com/miekg/dns
github.com/mitchellh/osext
github.com/pascaldekloe/goe
github.com/russross/blackfriday
github.com/russross/blackfriday/v2
github.com/seccomp/libseccomp-golang
gopkg.in/check.v1
gopkg.in/inf.v0
gopkg.in/tomb.v1

**Comment by wu-sheng** [comment]
How are those unknown going? Have you checked them manually? And maybe this could provide some ideas to enhance skywalking-eyes to identify those licenses too.

**Comment by wu-sheng** [comment]
Such as, I saw you have checked `github.com/pascaldekloe/goe`, which is CC0 License, which is in Catalog A. So it is fine.
But @kezhenxu94 @fgksgf the skywalking eyes don't identify it, but GitHub does. We should recheck about this.

**Comment by mrproliu** [comment]
> How are those unknown going? Have you checked them manually? And maybe this could provide some ideas to enhance skywalking-eyes to identify those licenses too.

The unknown licenses have been fixed in the PR. Some licenses could not be identified, I could submit some template for help skywalking-eyes to identity some licenses.

**Comment by wu-sheng** [comment]
> > How are those unknown going? Have you checked them manually? And maybe this could provide some ideas to enhance skywalking-eyes to identify those licenses too.
> 
> The unknown licenses have been fixed in the PR. Some licenses could not be identified, I could submit some template for help skywalking-eyes to identity some licenses.

Let's do that. Please submit an issue to track first.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**